services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py

```diff
@@ -42,6 +49,18 @@ async def _startup() -> None:
             app.state.resource_tracker_rabbitmq_consumer = await _subscribe_to_rabbitmq(
                 app
             )
+            # Setup periodic task
+            if app_settings.RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_CHECK_ENABLED:
+                lock_key = f"{app.title}:periodic_check_of_running_services_task_lock"
+                lock_value = "locked"
+                app.state.resource_tracker_background_task = start_periodic_task(
+                    exclusive(
```

Comment:
nice touch. How many replias you have of RUT?

Reply:
Currently, there are 3 (I think one would be sufficient but i want to test behaviour on more of them)